google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -41,22 +41,46 @@ public StorageClass apply(String constant) {
   private static final StringEnumType<StorageClass> type =
       new StringEnumType(StorageClass.class, CONSTRUCTOR);
 
-  /** Regional storage class. */
+  /**
+   * Regional storage class (deprecated).
+   * See:https://cloud.google.com/storage/docs/per-object-storage-class for details
+   */
   public static final StorageClass REGIONAL = type.createAndRegister("REGIONAL");
 
-  /** Multi-regional storage class. */
+  /**
+   * Multi-regional storage class (deprecated).
+   * See:https://cloud.google.com/storage/docs/per-object-storage-class for details
```

Comment:
This link redirects to https://cloud.google.com/storage/docs/storage-classes , so let's use that.

Reply:
done